app/controllers/patients_controller.rb

```diff
@@ -59,8 +59,30 @@ def edit
     @external_pledge = @patient.external_pledges.new
   end
 
+  # turned off until update is resolved
+  # def update_urgent_flag patient
+  #   if patient.pledge_sent?
+  #     patient.update_attributes urgent_flag: false
+  #   else
+  #     if patient.resolved_without_fund
+  #       patient.update_attributes urgent_flag: false
+  #     end
+  #   end
+  #   return patient
+  # end
+
   def update
     if @patient.update_attributes patient_params
+      # @patient = update_urgent_flag(patient)
+      @patient.reload
+      if @patient.pledge_sent?
```

Comment:
The core of this is pretty sound and looks good to me. I'm gonna hook this into an `after_save` callback on the Patient model -- seems dicey to me to put this in the controller layer, which is generally responsible for responding to http calls or whatever. (And probably add a test along the way!)